docs/en_US/guide/ai/python_openvino_tutorial.md

```diff
@@ -0,0 +1,286 @@
+# Running AI Algorithms with Python Function Plugins
+
+[LF Edge eKuiper](https://www.lfedge.org/projects/ekuiper/) is an edge lightweight IoT data analytics / streaming
+software which can be run at all kinds of resource constrained IoT devices.
+
+[OpenVINO](https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/overview.html) is an open source toolkit that makes it easier to write once, deploy anywhere.
+It converts and optimizes models trained using popular frameworks like TensorFlow, PyTorch, and Caffe. Deploy across a mix of Intel hardware and environments, on-premise and on-device, in the browser, or in the cloud.
+The example code and models for OpenVINO take reference from this [post](https://www.intel.cn/content/www/cn/zh/developer/articles/reference-implementation/industrial-surface-defect-detection.html), users can get more detail information from that link.
+
+By integrating eKuiper and OpenVINO, users can analyze the data more easily.
+In this tutorial, we will walk you through building an AI-enabled approach to segment defects from the surface powered by eKuiper and OpenVINO.
+The completed plugin package can be downloaded [here](https://github.com/lf-edge/ekuiper/blob/master/docs/resources/openvinoProject.zip), which also contains the full source code.
+
+## Prerequisite
+
+Before starting the tutorial, please prepare the following products or environments.
+
+1. Install the Python 3.x environment.
+2. Install the opencv-python, numpy and openvino packages via `pip install opencv-python==4.7.0.* openvino==2023.0.0 numpy==1.24.3`.
+
+By default, the portable plugin for eKuiper will run with the `python` command. If your environment does not support the `python` command, please use the [configuration file](../../configuration/global_configurations.md#portable-plugin-configurations) to modify the Python command, such as `python3`.
```

Comment:
```suggestion

```

Reply:
this one is still about the python environment, that's why I combined this one with Prerequisite 1